tests/main/interfaces-network-bind/task.yaml

```diff
@@ -31,6 +31,12 @@ prepare: |
 
     EOF
 
+restore: |
+    # This snap is removed because it generates thousands of DENIALS in the journal. Most of those
+    # are sent after the journalctl cursor for following test is determined producing errors while
+    # preparing the test.
+    snap remove network-bind-consumer
```

Comment:
Wondering if other tests leave potentially problematic snaps behind.

Reply:
@bboozzoo, I just saw this one, in fact the same snap in other tests is not causing problems like this.